Hi @sunu 

Now the error is showing about `Missing authorization headers`.
Basically, as a curator@apt.com account I followed this steps
- Started from scratch. `docker-compose down` and `docker-compose up db bootstrapper localstack api` 
- Clicked Download for document http://localhost:9000/documents/test-atbd-1/v2.0
  - v2.0 Journal PDF is failing due to missing `text` key. Maybe issue with bootstrap data
  - v2.0 Document PDF is showing `Missing authorization headers issue`
     - This is the link it gets forwared to [LINK](http://localhost:8000/v2/atbds/1/versions/v2.0/pdf?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6InRlc3Qta2V5In0.eyJleHAiOjE2Nzk5ODI4NzIsImlzcyI6Imh0dHA6Ly9sb2NhbGhvc3Q6NDU2Ni91cy1lYXN0LTFfNjc3MTY1NTZkMWExNDkxNTk5Y2FhNDI0NDc1ZmI1MzIiLCJzdWIiOiIxZTgyMWEyMC1iN2I5LTRjNjItOTM5Yi1mMDVmOTBlYzlmNGIiLCJhdXRoX3RpbWUiOjE2Nzk4OTY0NzIsImlhdCI6MTY3OTg5NjQ3MiwiZXZlbnRfaWQiOiJiNTE4OWZmMS00MjljLTRlZTktOGYzMy0zNzZjYjM4NWQ4NDYiLCJ0b2tlbl91c2UiOiJpZCIsImVtYWlsIjoiY3VyYXRvckBhcHQuY29tIiwiYXVkIjoiN2Qxd3NwYTZma2dmZWx0ZWZoaHNudmxiaXgiLCJjb2duaXRvOmdyb3VwcyI6WyJjdXJhdG9yIl0sImVtYWlsX3ZlcmlmaWVkIjoidHJ1ZSIsInByZWZlcnJlZF91c2VybmFtZSI6IkNhcmxvcyBDdXJhdG9yIiwiY29nbml0bzp1c2VybmFtZSI6IjFlODIxYTIwLWI3YjktNGM2Mi05MzliLWYwNWY5MGVjOWY0YiJ9.aeKYsS4Z4HhQHIsm7Gi8-0FfScdG84wJlNExIwMhw-bIiioqphAzAf042DzKQnVZJl6tXE1D32GZ1qwTy14XxOTWZ-3YkKIV4LXjn4c9Q0wpCvFSsSEWeLgleeAGV9cpYCvYNPoOcG8J7BfMYgNNwq4w1GDLWKbczle06PFmrITx1q8ExxCQrKFhNBedrUrX7WUhrGo2zPv59yOo-5RY1uA6h0irBWvTzjGQicKEsuQxDzkVm5Jm4oIPX0xJ51u9JwgpQRvmUjG2T2orwCh7gbLolVKssntBHeOB8U2GvbrbWZk_b87ui1Ubf2kFEfVXknopYyuihEkaJBfB7cbozQ) which has the token as GET param.

Also, typo in app/api/v2/pdf.py
https://github.com/NASA-IMPACT/nasa-apt/blob/29d185d98dc3ef69648a1fc543b5c710a67ccdea/app/api/v2/pdf.py#L174

Logs
For `text` key
```
backend-api-1           | During handling of the above exception, another exception occurred:
backend-api-1           |
backend-api-1           | Traceback (most recent call last):
backend-api-1           |   File "/app/./app/api/v2/pdf.py", line 155, in get_pdf
backend-api-1           |     local_pdf_filepath = generate_pdf(
backend-api-1           |   File "/app/./app/pdf/generator.py", line 982, in generate_pdf
backend-api-1           |     raise e
backend-api-1           |   File "/app/./app/pdf/generator.py", line 979, in generate_pdf
backend-api-1           |     latex_document = generate_latex(atbd, filepath, journal=journal)
backend-api-1           |   File "/app/./app/pdf/generator.py", line 713, in generate_latex
backend-api-1           |     output = prepare_sections.prepare_sections(
backend-api-1           |   File "/app/./app/pdf/prepare_sections.py", line 29, in prepare_sections
backend-api-1           |     "contents": fill_contents.fill_contents(
backend-api-1           |   File "/app/./app/pdf_utils/fill_contents.py", line 67, in fill_contents
backend-api-1           |     contents.append(process_lists.ul_ol_lists(element))
backend-api-1           |   File "/app/./app/pdf_utils/process_lists.py", line 29, in ul_ol_lists
backend-api-1           |     for item in list_items(child):
backend-api-1           |   File "/app/./app/pdf_utils/process_lists.py", line 18, in list_items
backend-api-1           |     return [d["text"] for _indx, d in enumerate(items)]
backend-api-1           |   File "/app/./app/pdf_utils/process_lists.py", line 18, in <listcomp>
backend-api-1           |     return [d["text"] for _indx, d in enumerate(items)]
backend-api-1           | KeyError: 'text'
```

--- a/app/api/v2/pdf.py
+++ b/app/api/v2/pdf.py
@@ -171,7 +171,7 @@ def get_pdf(
                         access_token = request.headers.get("x-access-token", "")
                         if not id_token or not access_token:
                             return JSONResponse(
-                                satus_code=400,
+                                status_code=400,
                                 content={
                                     "message": "Missing authorization headers. "
                                     "Please include the 'authorization' and 'x-access-token' headers in your request."

